idunn/utils/prometheus.py

```diff
@@ -0,0 +1,47 @@
+import time
+import contextlib
+
+from apistar import Component
+from prometheus_client import Counter, Gauge, Histogram
+
+
+""" The logic of the Prometheus metrics is defined in this module """
+
+
+IDUNN_WIKI_REQUEST_DURATION = Histogram(
+    "idunn_wiki_request_duration_seconds",
+    "Time spent processing a Wiki request.",
+    ["target", "handler"],
+)
+
+IDUNN_WIKI_EXCEPTIONS_COUNT = Counter(
+    "idunn_exceptions_count",
```

Comment:
aren't them all about wikipedia ? if so it should be in the name, else you should change the name of the variable

Reply:
done